main/templates/main/index.html

```diff
@@ -65,34 +63,51 @@
   <a class="home-button" href="{% url 'our_team' %}">Our awesome team!</a>
 </section>
 
+<!--Projects-->
 <section id="section-projects">
-  <div class="container">
-    <h2>Our Projects</h2>
-    <div class="row">
-    {% for project in projects %}
-      <div class="col-sm-4">
-        <a href="{% url 'project_details' project.id %}">
-          <div class="project-card">
-            {% if project.image %}
-              <img src="{{project.image.url}}" alt="{{project.title}}" class="project-pic">
-            {% endif %}
-            <div class="project-text">
-              <div class="project-title">
-                {{project.title}}
-              </div>
-              <div class="project-short-desc">
-                {{project.shortDesc}}
+  <a name="projects">
```

Comment:
This is actually the old way of making jump links, which isn't HTML5 complaint and is being phased out.

The new way is to just put the div ID in the # section of the URL. It'd be cleaner if we could change to that, though it isn't necessary or urgent (all major browsers support both and will keep supporting both for the near future).